.github/stale.yml

```diff
@@ -0,0 +1,33 @@
+# Stale Issue & PR Rules
+#
+# See https://github.com/probot/stale#usage for values
+#
+
+
+
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+
+# only issues with these labesl will be considered ([] is disabling this rule, look at all issues except excempt)
+onlyLabels: []
+
+
+# Label to use when marking an issue as stale
+staleLabel: stale
+
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```

Comment:
What do you think of exempting issues with `bug`, `confirmed` and `enhancement`?

```
issues:
  exemptLabels:
    - confirmed
    - bug
    - enhancement
```